app/models/job_invocation.rb

```diff
@@ -139,6 +139,10 @@ def to_label
     description
   end
 
+  def latest_jobs(limit = 5)
+    JobInvocation.order(id: :desc).limit(limit)
```

Comment:
Food for thought: Do we want last N jobs by time when they were submitted (and therefore by id) or by start_at/started_at?

Reply:
Hm, interesting :-) I'd ask the reporter :-)

Reply:
Good catch :+1: jobs are now sorted by time